frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -180,6 +185,24 @@ const plugin: Plugin<ConsumedExtensions> = [
       },
     },
   },
+  {
+    type: 'Page/Route',
+    properties: {
+      exact: true,
+      path: `/k8s/ns/:ns/persistentvolumeclaims/:appName/volumesnapshots/:name`,
+      loader: () =>
+        import(
+          './components/volume-snapshot/volume-snapshot' /* webpackChunkName: "metal3-baremetalhost" */
+        ).then((m) => m.VolumeSnapshotDetails),
```

Comment:
Please make the file export it as the default export

Reply:
File has multiple pages to export, that's why not using default here.